canary/producer-c/canary/KvsProducerSampleCloudwatch.cpp

```diff
@@ -197,8 +197,10 @@ INT32 main(INT32 argc, CHAR* argv[])
     BOOL cleanUpDone = FALSE;
     BOOL fileLoggingEnabled = FALSE;
     BOOL runTillStopped = FALSE;
-
+    PAuthCallbacks pAuthCallbacks = NULL;
     CanaryConfig config;
+    BOOL firstFrame = TRUE;
+    UINT64 startTime = 0;
```

Comment:
Do we need to init startTime here? Some static analysis tools will generate warnings for this as its value gets overwritten later with GETTIME(). It's a minor issue so feel free to ignore.

Reply:
I usually have the habit of assigning an initial value to a variable. I can remove this.